README.md

```diff
@@ -85,7 +87,7 @@ most one such item per line. This allows the git notes to be automatically
 merged using the "cat\_sort\_uniq" strategy.
 
 Since these notes are not in a human-friendly form, all of the refs used to
-track them start with the prefix "refs/notes/devtools". This helps make it
+track them start with the prefix "refs/notes/git-review". This helps make it
```

Comment:
if this tool is called `git-appraise`, or `git-review` as with my suggestion, then take ownership over the git-appraise/git-review namespace, not `devtools`